runtime/gc_modron_startup/mmparseXXgc.cpp

```diff
@@ -716,6 +716,143 @@ gcParseXXgcArguments(J9JavaVM *vm, char *optArg)
 		}
 
 #if defined(J9VM_GC_MODRON_SCAVENGER)
+		/* Start of options relating to dynamicBreadthFirstScanOrdering */
+		if(try_scan(&scan_start, "dbfGcCountBetweenHotFieldSort=")) {
+			UDATA value;
+			if(!scan_udata_helper(vm, &scan_start, &value, "dbfGcCountBetweenHotFieldSort=")) {
+				returnValue = JNI_EINVAL;
+				break;
+			}
+			if(value > 10) {
+				j9nls_printf(PORTLIB, J9NLS_ERROR, J9NLS_GC_OPTIONS_INTEGER_OUT_OF_RANGE, "dbfGcCountBetweenHotFieldSort=", (UDATA)0, (UDATA)10);
+				returnValue = JNI_EINVAL;
+				break;
+			}
+			extensions->gcCountBetweenHotFieldSort = ((int)value);
```

Comment:
Cast to `(int)` is not necessary.  The type of `extensions->gcCountBetweenHotFieldSort` is `uintptr_t`